card.go

```diff
@@ -163,6 +175,12 @@ func get(ix any, k string) any {
 func (c *Card) dump() CardD {
 	data := make(map[string]any)
 	var bufs []BufD
+
+	// Look for nested buffers within form_card.
+	if v, ok := c.data["view"]; ok && v.(string) == "form" {
+		fillFormCardBufs(c.data, data, &bufs, make([]string, 0))
```

Comment:
At first I was puzzled with `make([]string, 0))` but now I see the function is recursive. Seems ok.